libparsec/src/path.rs

```diff
@@ -0,0 +1,23 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use libparsec_types::{EntryName, FsPath};
+
+pub fn path_join(root: &FsPath, leaf: &EntryName) -> FsPath {
```

Comment:
A `directory` can be assimilate as a tree, not `path`
```suggestion
pub fn path_join(parent: &FsPath, child: &EntryName) -> FsPath {
```